docker-compose/cse-valid-d2cc/config/data-bridge-d2cc-crac-export-corner.yml

```diff
@@ -0,0 +1,25 @@
+data-bridge:
+  zone-id: "Europe/Paris"
+  target-process: CSE_VALID_D2CC
+  file-type: CRAC
```

Comment:
file-type has to be unique so it should not be the same for import-corner CRAC and export-corner CRAC. Maybe rename IMPORT_CRAC and EXPORT_CRAC